modules/microshift-auto-apply-manifests.adoc

```diff
@@ -76,22 +74,22 @@ spec:
         command:
           - sleep
           - "3600"
+EOF
 ----
-// what are the "---" after name: busybox?
 .Procedure
-. Create the `kustomize` manifest files by running the following commands:
+. Create the `kustomize` manifest files by running the following commands.
 +
-* Run the command to read and write the standard inputs.
+.. Run the command to read and write the standard inputs:
 [source,terminal]
 +
 ----
 $ sudo tee
 ----
-* Apply the `yaml` configuration.
+.. Apply the `yaml` configuration:
 +
 [source,terminal]
 ----
-$ {MANIFEST_DIR}/kustomization.yaml &>/dev/null
+$ oc apply {MANIFEST_DIR}/kustomization.yaml
```

Comment:
Could you please use the `cat` instead of `tee` here (to be consistent with the above step creating busybox.yaml), for example:
```
sudo cat << EOF > ${MANIFEST_DIR}/kustomization.yaml
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
namespace: busybox
resources:
  - busybox.yaml
images:
  - name: BUSYBOX_IMAGE
    newName: registry.k8s.io/busybox
EOF
```